### PR TITLE
Afterburner connector cover for 42STH20-1004AS

### PR DESCRIPTION
**Pull request by hhammarstrand**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
A connector cover for 42STH20-1004AS to use with the Afterburner

#### How was it tested? 
Tested by me and a few users on the discord. I'm not sure it has been tested on 1.8 or 2.4, but it has been tested on the switchwire (which is missing in the compatibility table here in voronusers?)

#### Any background context you want to provide?
I have been asked for this connector cover multiple times on discord, that's why I'm submitting this, to have a place with all the info and latest versions. 

#### Screenshots (if appropriate)
compare

#### Further notes
Wasn't sure if this did count as: 

*The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*

After contacting a admin on discord I was told to give it a try to submit.

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod.  Can you rename your `readme.md` to `README.md` for consistency with other mods?  Also, can you modify the top level `printer_mods/README.md` to add your mod to the table in alphabetical order?  Everything else looks good.

Checked:
- mod in correct directory
- no spaces in directory names
- there's a readme for the mod
- STL is oriented correctly and doesn't require supports

File: printer_mods/README.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ Update the following table with the information about your mod:
 
 like so:
 `
-| Creator | [Mod title](link) | Description | :x: | :x: | :x: |`
+| Creator | [Mod title](link) | Description | :x: | :x: | :x: | :x |`
```
Comment (KiloQubit) [inline printer_mods/README.md:22]: missing second colon - change `:x` to `:x:` .

File: printer_mods/README.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ like so:
 | |[Short Z Joints](./Ellis/Short_Z_Joints) | Shorter Z joints for the v2.4 |:x: |:x: |:heavy_check_mark: |
 | FreeformFez | [V1.8 Filtered Power Inlet](./FreeformFez/FilteredPower) | V1.6 Filtered Inlet and Rocker Switch for V1.8 | :x: | :heavy_check_mark: | :x: |
 | GSL12 | [Pocketwatch Nema 17](./GSL12/pocketwatch_nema17)| Extruder like Pocketwatch, but with Nema17 instead of Nema14. Maybe for V0 fork with 2020 extrusions, Nema17 and MGN9 or other printers. | :grey_question: | :grey_question: | :grey_question: |
+| hhammarstrand | [Afterburner connector cover for 42STH20-1004AS](./hhammarstrand/afterburner_connector_cover_42STH20-1004AS) | Connector cover for 42STH20-1004AS | :x: | :grey_question| | :grey_question | :heavy_check_mark: |
```
Comment (KiloQubit) [inline printer_mods/README.md:48]: missing second colon - change both `:grey_question` to `:grey_question:`

also you have an extra `|` between the two grey questions, which breaks the table

**Review by KiloQubit — CHANGES_REQUESTED** [review]

**Review by KiloQubit — APPROVED** [review]
thanks, looks good!

**Comment by andrey-kk** [comment]
Sorry but this mod does not work with IGUS chain. It covers LDO stepper well but goes too far to the back and touches chain. But ns home position where the chain ends it evel locks the cartridge and doe not let go back.
Tested on v2.4 with LDO extrruder stepper. For now have not found good option as stock one does not work either.

**Comment by hhammarstrand** [comment]
> Sorry but this mod does not work with IGUS chain. It covers LDO stepper well but goes too far to the back and touches chain. But ns home position where the chain ends it evel locks the cartridge and doe not let go back.
> 
> Tested on v2.4 with LDO extrruder stepper. For now have not found good option as stock one does not work either.

Hi, thanks for your feedback! 
As stated it has only been tested with the switchwire since I don't have a 2.4 yet. 

Would you mind sharing some pictures so that I can see if it's possible to adapt it for the 2.4? 

I'm @hhammarstrand @ discord so feel free to reach out so that we might find a solution together. 

Have a nice evening!

**Comment by andrey-kk** [comment]
Hi, unfortunately cannot make picture now as I am still printing the full set after assembled pif setup. Was fighting with faulty heater and thermistor, only today fixed.
Will do my measurementrs and provide feedback or even design a modded part in fusion. In short- the bottom screw mount and part going down from it scratches igus chain as too close. Mod is needed so that overal length of the cover is like stock one but only screw mounts are extended. Your reference point is igus chain mount. It's lover part goes exactly over igus chain so you do not go lower then it before you reach the postion of original cover as it is measured to be just after igus chain not touching it.

**Review by raymondh2 — APPROVED** [review]

**Comment by FHeilmann** [comment]
@hhammarstrand in light of these reports, please replace the `:grey_question:` entries in your readme row with `:x:` to indicate incompatibility

**Comment by hhammarstrand** [comment]
> @hhammarstrand in light of these reports, please replace the `:grey_question:` entries in your readme row with `:x:` to indicate incompatibility

Fixed!

**Comment by psun786** [comment]
I tried to open the step file but it shows corrupted. Any idea? Thanks!

**Pull request merged**